parsec/README.md

```diff
@@ -1,34 +1,130 @@
 # parsec [![package publisher](https://img.shields.io/pub/publisher/parsec.svg)](https://pub.dev/packages/parsec/publisher) [![pub package](https://img.shields.io/pub/v/parsec.svg)](https://pub.dev/packages/parsec)
 
-The multi-platform `parsec` plugin for Flutter to calculate math equations using C++ library.
+The multi-platform `parsec` plugin for Flutter to calculate math equations using C++ library with **Smart Load Balancing** for optimal performance.
+
+## 🚀 Smart Load Balancer Architecture
+
+**parsec** now features an intelligent routing system that automatically selects the optimal evaluation method based on equation content and platform:
+
+### **Performance Optimization**
+- **Web Platform + Standard Equations** → WebAssembly via [parsec-web](https://github.com/oxeanbits/parsec-web): **~1ms** ⚡
+- **Web Platform + Custom Functions** → Backend Server: ~110ms (database access required)
+- **Native Platform + Standard Equations** → Method Channels: ~5-10ms
+- **Native Platform + Custom Functions** → Backend Server: ~110ms (database access required)
+
+### **Automatic Custom Function Detection**
```

Comment:
Does not make sense to be inside a public library, because these custom functions only exists in the private systems. An idea would be to expose a way to pass to parsec initialization, the list of custom functions